src/k8s_sandbox/_sandbox_environment.py

```diff
@@ -10,6 +10,7 @@
     SandboxEnvironmentConfigType,
     sandboxenv,
 )
+from inspect_ai.util._sandbox import SandboxConnection
```

Comment:
```suggestion
from inspect_ai.util import SandboxConnection
```
https://github.com/UKGovernmentBEIS/inspect_ai/blob/4d174288d3d723d6146213faee86d03334c9fa62/src/inspect_ai/util/__init__.py#L11

Based on the import here, I'm betting this will just work